analytics-integrations/kahuna/src/main/java/com/segment/analytics/internal/integrations/KahunaIntegration.java

```diff
@@ -83,4 +149,14 @@
 
     KahunaAnalytics.logout();
   }
+
+  @Override public void alias(AliasPayload alias) {
+    KahunaAnalytics.setUserCredential(USER_ID_KEY, alias.userId());
+  }
+
+  private static String getISO8601StringForDate(Date date) {
```

Comment:
You can reuse this method - https://github.com/segmentio/analytics-android/blob/master/analytics-core/src/main/java/com/segment/analytics/internal/Utils.java#L71 instead.